### ISSUE TITLE
Update node-notifier in version 5

### ISSUE THREAD

**Issue by range-of-motion** [JeffreyWay/laravel-mix#2690]
-   Laravel Mix Version: 5.0.9 (`npm list --depth=0`)
-   Node Version (`node -v`): irrelevant
-   NPM Version (`npm -v`): irrelevant
-   OS: irrelevant

### Description:

I'm using the latest version of Laravel Mix v5.x, but have recently gotten a warning from GitHub informing me about a vulnerable package (node-notifier)–see screenshot below. 

I'm unable to update said package because Laravel Mix (at least in v5.x) requires v6.x of node-notifier (the vulnerable package), whereas v8.0.1 is the earliest version with a fix.

TLDR–would it be possible to patch node-notifier to v8.0.1 or higher in Laravel Mix v5.x?

![image](https://user-images.githubusercontent.com/9268822/102825130-7c2c2280-43de-11eb-9a29-85045f6039a7.png)

**Comment by range-of-motion** [comment]
I'm just now seeing that this is due to `webpack-notifier` being "stuck" at v1.5.x.

**Comment by range-of-motion** [comment]
@JeffreyWay bumping in case you haven't seen this.
